vitest.config.ts

```diff
@@ -0,0 +1,19 @@
+import { defineConfig } from 'vitest/config';
+import AutoImport from 'unplugin-auto-import/vite';
+
+export default defineConfig(({ mode }) => ({
+  resolve: {
+    conditions: [],
+  },
+  test: {
+    globals: true,
+    environment: 'jsdom',
+    setupFiles: ['./vitest-setup.ts'],
+  },
+  plugins: [
+    AutoImport({
```

Comment:
As you are using globals (L9), I think there is no need for `unplugin-auto-import`